docs/README.md

```diff
@@ -136,147 +136,154 @@ Find below the needed instructions to perform the User Identifier Extension in y
   const onSubmit = useCallback(async () => {
-    const email = `${inputValue}@email.com`
-    return email
-  }, [inputValue])
+    const email = `${inputValue}@email.com`;
+    return email;
+  }, [inputValue]);
 
-  // This code registers the async callback function you defined in the login app when this component mounts.
+  // This code adds the async callback function you defined in the login app when this component mounts.
   useEffect(() => {
-    registerSubmitter(onSubmit)
-  }, [registerSubmitter, onSubmit])
+    registerSubmitter(onSubmit);
+  }, [registerSubmitter, onSubmit]);
 
   // The component calls "renderInput" and renders its result.
   return renderInput({
     value: inputValue,
     onChange: onChangeInput,
     placeholder: identifierPlaceholder,
-  })
-}
+  });
+};
 
-export default ComponentName
+export default ComponentName;
 ```
 
-3. In the `store/interfaces.json` file, create a new [interface](https://vtex.io/docs/getting-started/desenvolva-componentes-usando-vtex-io-e-react/4#entendendo-interfaces) for `user-identifier`, declaring the component you created in the previous step:
+3. In the `store/interfaces.json` file, create a new [interface](https://developers.vtex.com/docs/guides/vtex-io-documentation-interface) for `user-identifier`, declaring the component you created in the previous step:
 
 ```json
 "user-identifier.{{InterfaceName}}": {
   "component": "{{ComponentName}}"
 },
 ```
 
-> ℹ️ *The interface name, which is represented in the example above as `{{InterfaceName}}`, can be any of your choosing.* 
-> ℹ️ *In the example above, the component created in the step 2 is represented by `{{ComponentName}}`.*
+> ℹ️ _The interface name, which is represented in the example above as `{{InterfaceName}}` can be anything your want._ ℹ️ _In the example above, the component created in step 2 is represented by `{{ComponentName}}`._
 
-4. In the `store/plugins.json` file, plug in the new interface (`user-identifier`) to the Login app:
+4. In the `store/plugins.json` file, add the new interface (`user-identifier`) for the Login app:
 
 ```json
 "login > user-identifier": "user-identifier.{{InterfaceName}}",
 "login-content > user-identifier": "user-identifier.{{InterfaceName}}",
 ```
 
-> ℹ️ *If the `store/plugins.json` file is not available, you should create it.*
+> ℹ️ _If the `store/plugins.json` file is not available, you should create it._
 
-> ⚠️ *Since the Login app behaves as if users were typing their emails, error messages like `The email is invalid` may appear even if the component is asking users other identifiers. This message, along with others, can be edited in the admin's CMS.*
+> ⚠️ _Since the Login app behaves as if users were typing their emails, error messages like `The email is invalid` may appear even if the component is requesting other identifiers from users. This message, and others, can be edited in the Admin CMS._
```

Comment:
```suggestion
> ⚠️ Since the Login app behaves as if users were typing their emails, error messages like `The email is invalid` may appear even if the component is requesting other identifiers from users. This message, and others, can be edited via the Store Editor in the Admin.
```